### PR TITLE
Update some eclasses for EAPI 8

### PR DESCRIPTION
**Pull request by ulm**

### CODE REVIEW

File: eclass/autotools.eclass
Comment (SoapGentoo) [inline eclass/autotools.eclass:34]: this needs to be put on hold, there's some other issues that need to be resolved first

File: eclass/git-r3.eclass
Comment (SoapGentoo) [inline eclass/git-r3.eclass:26]: I mean, I get it, but is this really such a big win? not that you should remove it, but it increases cyclomatic complexity purely for a syntactic sugar?

File: eclass/gnuconfig.eclass
Comment (SoapGentoo) [inline eclass/gnuconfig.eclass:20]: needs signoff from @thesamesam

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: eclass/git-r3.eclass
Comment (ulm) [reply eclass/git-r3.eclass:26]: It's mainly a reminder for removal of the old `+=` assignment later.

File: eclass/gnuconfig.eclass
Comment (thesamesam) [inline eclass/gnuconfig.eclass:23]: While here:
```suggestion
		die "EAPI ${EAPI:-0} is unsupported!"
```

**Review by thesamesam — APPROVED** [review]

File: eclass/git-r3.eclass
Comment (ulm) [reply eclass/git-r3.eclass:26]: Dropped.

File: eclass/libtool.eclass
Comment (ulm) [reply eclass/libtool.eclass:25]: That doesn't look it would need to be blocked on EAPI update.

Comment (thesamesam) [reply eclass/libtool.eclass:25]: Sure.

**Review by thesamesam — APPROVED** [review]

**Pull request closed without merging**
